fv/ipip_test.go

```diff
@@ -323,7 +323,10 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ IPIP topology before adding
 	})
 
 	Context("after removing BGP address from nodes", func() {
-		// Simulate having a host send IPIP traffic from an unknown source, should get blocked.
+		// After removing BGP address from nodes, the nodeAddress should be used.
+		// Hence we expect the IPSets to have the HostIPs of Felix[0], Felix[1] in case of IPTables
+		// In BPF mode, the route should not be removed for Felix[0].IP and workload IP
+		// Connectivity between workloads should still work.
```

Comment:
The test is no longer verifying the IPIP source policing and I think we need to verify that.  Could we delete the Node entirely, or change its IP in the datamodel to be wrong so that the IP set no longer has the right value in it (for example)?